index.html

```diff
@@ -1,21 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="index.css">
-    <script src="./main.js" defer></script>
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/adf6e8ca6b.js" crossorigin="anonymous"></script>
+  <title>Awesome books</title>
+  <link rel="stylesheet" href="./main.css">
 </head>
 <body>
-    <div id="books">
-     <h1>Awesome Books</h1>
-     <ul></ul>
-     <button id="remove">remove all</button><br><br><br>
-    </div>
-    <label for="Title">Title:</label><input type="text" id="title" placeholder="Title" required><br><br>
-    <label for="Author">Author:</label><input type="text" id="author" placeholder="Author" required><br><br>
-    <button id="add" type="submit">add</button><br><br>
-</body>
+  
+  <nav class="nav-bar">
+    <p id="logo">Awesome Books</p>
+    <ul class="nav-list">
+      <li class="nav-item" id="list">List </li>
+      <li>|</li>
+      <li class="nav-item" id="addNew">Add new</li>
+      <li>|</li>
+      <li class="nav-item" id="contact">Contact</li>
+    </ul>
+  </nav>
```

Comment:
- [ ] Please let us implement the time and date details at the top-right just below the navigation as shown in the wireframe. For this, you can use the [Luxon](https://moment.github.io/luxon/docs/manual/install.html) library. See [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time) 😎 
### Wireframe
![image](https://github.com/beyk/Awesome_Book3/assets/46839548/9527b341-6be8-4aba-8473-225778e853e3)
### Your Implementation
![image](https://github.com/beyk/Awesome_Book3/assets/46839548/def5e10d-2418-42f5-8cc9-dcf02fe162d3)